sections/05_treat.md

```diff
@@ -1,35 +1,42 @@
 ## The impact of deep learning in treating disease and developing new treatments
 
 `TODO: write Treat intro`
+
+`TODO: just realised this wanders heavily into the categorization area. Need some high-level rearrangment? Could shift the first section below into the categorization section and leave the treatment here.`
+
+Given the ever-present need to make better interventions faster at the point of care -- incorporating the complex calculus of a patients
+symptoms, diagnostics and life history -- there is a long history of attempts to
+apply deep learning to the problem of treatment. Success in this area would also be very useful for directions like personalised healthcare or precision medicine [@doi:10.1056/NEJMp1006304 @doi:10.1155/2013/769639]. Broadly, these may divided into
+methods for improved classification of patients (i.e. improved and those for development of
```

Comment:
I think this linkage is nice - can you refer to the earlier section about categorizing patients? Otherwise it looks like we're not quite aware of what's in our own review. ;)